linopy/common.py

```diff
@@ -316,7 +318,7 @@ def check_has_nulls_polars(df: pl.DataFrame, name: str = "") -> None:
         ValueError: If the DataFrame contains null values,
         a ValueError is raised with a message indicating the name of the constraint and the fields containing null values.
     """
-    has_nulls = df.select(pl.col("*").is_null().any())
+    has_nulls = df.select(pl.col("*").is_null().any()).collect()
```

Comment:
perhaps, we can also avoid this `.collect`?

Reply:
Yes, we should be able to, but i think we need to change the formulation then a bit further.